dts/bindings/mtd/partition.yaml

```diff
@@ -28,3 +28,7 @@ child-binding:
           type: array
           description: register space
           required: true
+       fstab-entry:
```

Comment:
can we do this the other way around?  meaning the fstab entry has a phandle to the mtd partition?

Reply:
We do.  That property is obsolete and needs to be removed.